src/layer/unitary.rs

```diff
@@ -100,30 +100,71 @@ fn to_real(ar: Array) -> Array {
     af::join(1, &af::real(&ar), &af::imag(&ar))
 }
 
+fn mod_relu(z: Array, b: Array) -> Array {
```

Comment:
Can these be moved to activations? Just don't add them to `activations::get_activation` & `activations::get_activation_derivative` because of the extra parameter